tests/e2e/util.go

```diff
@@ -1948,6 +1969,38 @@ func replacePasswordRotationTime(file, host string) error {
 	return nil
 }
 
+// performPasswordRotationOnSupervisor invokes the given command to replace the password
+//
+//	Vmon-cli is used to restart the wcp service after changing the time.
+func performPasswordRotationOnSupervisor(supervisor_id string, host string) error {
+	sshCmd := fmt.Sprintf("vmon-cli --stop %s", wcpServiceName)
+	framework.Logf("Invoking command %v on vCenter host %v", sshCmd, host)
+	result, err := fssh.SSH(sshCmd, host, framework.TestContext.Provider)
+	time.Sleep(sleepTimeOut)
+	if err != nil || result.Code != 0 {
+		fssh.LogResult(result)
+		return fmt.Errorf("couldn't execute command: %s on vCenter host: %v", sshCmd, err)
+	}
+
+	sshCmd = fmt.Sprintf("cd /etc/vmware/wcp; sudo -u wcp psql -U wcpuser -d VCDB -c \"update cluster_db_configs set last_storage_pwd_rotation_timestamp=1684717716 where instance_id='%s'\"", supervisor_id)
```

Comment:
can you explain the logic around
`last_storage_pwd_rotation_timestamp=1684717716`

Reply:
we don't need to hard code it then right